components/Faq.js

```diff
@@ -1,7 +1,130 @@
+
+const infoArray = [
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+]; // dummy data
+
+const colors = [
+	"#82aaff",
+	"#c792ea",
+	"#ffd700",
+	"#c3e88d",
+    "#ff5370",
+	"#f07178",
+	"#89ddff",
+	"#f78c6c",
+	"#80cbc4",
+	"#da70d6",
+]
+
+const colorArray = [
+	`bg-color1`,
+	`bg-color2`,
+	`bg-color3`,
+	`bg-color4`,
+	`bg-color5`,
+	`bg-color6`,
+	`bg-color7`,
+	`bg-color8`,
+	`bg-color9`,
+	`bg-color10`,
+];
+
+const FaqItem = props => {
+	colorCount = props.index;
+	if (colorCount == 10) {
+		colorCount = 1;
+		console.log("Color reset to 1!");
+	}
+
+	let bgClass = "bg-white";
+	let bgStyle = {};
+
+	if(props.index != 0 )
+	{
+		bgClass = `bg-color${props.index}`;
+	}
+
+	if(props.index == infoArray.length-1)
+	{
+		bgClass = '';
+		bgStyle = {
+			boxShadow: `0px -5vw ${colors[infoArray.length-2]}`,
+		};
+		console.log("Last element: " + bgStyle);
+	}
+
+	return (
+		<Accordion.Item className={`${bgClass} font-lexend m-0 p-0`} id={`element${props.index}`} eventKey={props.index} >
+			<Accordion.Button className={`bg-black text-white rounded-t-lg -mt-[0.4vw]`}>
+				Lorem ipsum dolor sit amet? 
```

Comment:
This should be populated with a question from props

Reply:
Updated in latest commit!